lib/iframe-browser.js

```diff
@@ -90,6 +92,27 @@ define(function (require, exports, module) {
         return document.getElementById("bramble-iframe-browser");
     }
 
+    // Open a new window with the content of the iframe
+    // Only 1 window is allowed for the preview
+    function detachPreview() {
+        var iframe = getBrowserIframe();
+
+        if(!iframe) {
+            return;
+        }
+
+        var currentURL = getBrowserIframe().src;
```

Comment:
`var currentURL = iframe.src;` --you already have the iframe cached in the local variable.